chains/ethereum/src/client.ts

```diff
@@ -15,6 +15,8 @@ import type { ConnectorId } from './types'
 
 export class EthereumClient {
   private readonly wagmi = {} as Client
+  public walletConnectUri = ''
+  public walletConnectVersion = 1
```

Comment:
Shouldn't this be `'1'`?

Reply:
Kept is as number internally, but it is string in wagmi.